dev/issues.rst

```diff
@@ -16,38 +16,58 @@ Examples include "Duplicate of #123", "Discovered to be configuration
 error", "Rendered moot by #123" and so on. We don't use the "invalid" or
 "wontfix" labels.
 
+api
+    Issues related to the API and configuration.
+
 bug
-    The issue is a verified bug.
+    Issues verified to be a bug.
```

Comment:
```suggestion
    A problem with current functionality, as opposed to missing functionality (enhancement).
```

(The "verified" part doesn't apply any more since we use the needs-triage label for that)